docs/README.md

```diff
@@ -0,0 +1,97 @@
+Documentation Index
+===================
+
+This page serves as a categorized index for all `kops` documentation.
+
+
+Quick start
+-----------
+* [Getting started on AWS](aws.md)
+* [CLI reference](cli/kops.md)
+
+
+Overview
+--------
+
+* [CLI](#cli)
+* [Introspection](#introspection)
+* [`kops` design documents](#kops-design-documents)
+* [Networking](#networking)
+* [Operations](#operations)
+* [Security](#security)
+* [Workflows](#workflows)
+
+
+
+### CLI ###
+
+* [CLI argument explanations](arguments.md)
+* [CLI reference](cli/kops.md)
+* [Commands](commands.md)
+    * miscellaneous CLI-related remarks
+* [Experimental features](experimental.md)
+    * list of and how to enable experimental flags in the CLI
+* [kubectl](kubectl.md)
+    * how to point kubectl to your `kops` cluster
+
+
+### Introspection ###
+
+* [Download `kops` configuration](download_config.md)
+    * methods to download the current generated `kops` configuration
+* [Get AWS subdomain NS records](ns.md)
+
+
+### `kops` design documents ###
+
+* [`kops` cluster boot sequence](boot-sequence.md)
+* [`kops` cluster spec](cluster_spec.md)
+* [`kops` instance groups](instance_groups.md)
+* [`kops` philosophy](philosophy.md)
+* [`kops` state store](state.md)
+
+
+### Networking ###
+
+* [Networking overview](networking.md)
+* [Run `kops` in an existing VPC](run_in_existing_vpc.md)
+* [Supported network topologies](topology.md)
+* [Subdomain setup](creating_subdomain.md)
+
+
+### Operations ###
+
+* [Cluster addon manager](addon_manager.md)
+* [Cluster addons](addons.md)
+* [Cluster configuration management](changing_configuration.md)
+* [Cluster with HA creation example](advanced_create.md)
+* [Cluster upgrades and migrations](cluster_upgrades_and_migrations.md)
+* [`etcd` volume encryption setup](etcd_volume_encryption.md)
+* [`etcd` backup setup](etcd_backup.md)
+* [GPU setup](gpu.md)
+* [High Availability](high_availability.md)
+* [InstanceGroup images](images.md)
+    * how to modify IG images and information on available/tested images
```

Comment:
Don't you think that if by the time a user wants to use custom IG images he knows what an IG is? It's also named in the "kops design documents" list, so a Ctrl+F would find it.